install/kubernetes/helm/istio/values-istio-grafana-config.yaml

```diff
@@ -0,0 +1,544 @@
+# Top level istio values file has the following sections.
```

Comment:
Was this accidentally added? I am not sure what this is for

Reply:
If you need to add something to the values for this you can just add it to the `grafana/values.yaml` file

Reply:
Hi @howardjohn, thanks for the reply. I didn't want to go an edit the sub chart values directly. I was hoping to create a single values.yaml file when runing the `helm install -f ...`

Reply:
I believe I added this file as an example of how to the use env config with grafana.